test/hls_reference/video_mixer/index.m3u8

```diff
@@ -1,5 +1,5 @@
 #EXTM3U
 #EXT-X-VERSION:7
 #EXT-X-INDEPENDENT-SEGMENTS
-#EXT-X-STREAM-INF:BANDWIDTH=727705,CODECS="avc1.42e00a"
+#EXT-X-STREAM-INF:BANDWIDTH=249884,CODECS="avc1.42e00a"
```

Comment:
Suspicious :thinking:

Reply:
I also only can suspect that this can be related to BlankVideoGenerator.

Reply:
That;s probably because of different segment durations so the longer segment has lower avg bitrate